src/components/reservation/UserListItems.tsx

```diff
@@ -84,7 +79,10 @@ const UserListItems: FC<UserListItemsProps> = ({
             buttonAriaLabel={t("changeInterestPeriodText")}
           />
           <NoInterestAfterModal
-            selectedInterest={selectedInterest ?? defaultInterestPeriod}
+            selectedInterest={
+              selectedInterest ??
+              Number(interestPeriods.defaultInterestPeriod.value)
```

Comment:
Isn't there a way to make sure that `interestPeriods.defaultInterestPeriod.value` is a number in an earlier stage. I think we should avoid casting values if possible.

Reply:
Unfortunately the `Option` type is defined with a `value: string`, and we use this type in multiple dropdowns in the reservation workflow, so changing it there wouldn't be smart, I'd say. But I could alter the <NoInterestAfterModal> component! That would be a solution I think :)